tpay_docker/Ubuntu_18.04/Dockerfile

```diff
@@ -0,0 +1,97 @@
+FROM ubuntu:18.04
+
+#dependencies
+RUN apt-get update
+RUN apt-get install -y build-essential \
+     libtool autotools-dev automake \
+     pkg-config zlib1g-dev libevent-dev \
+     bsdmainutils git libboost-all-dev \
+     libseccomp-dev libcap-dev libminiupnpc-dev \
+     libqt5gui5 libqt5core5a libqt5webkit5-dev \
+     libqt5dbus5 qttools5-dev qttools5-dev-tools \
+     libprotobuf-dev protobuf-compiler libqrencode-dev \
+     libssl-dev qt5-default
+RUN apt-get install -y wget
+
+#openssl
```

Comment:
Is this necessary? Starting from Ubuntu 18.04, `libssl` comes with version `1.1.0`

Reply:
I wasn't aware of it, i will remove this and test it this evening, thanks.